Maria-Edgeworth-Letters-Project/me-tei
# Add refs to persName, remove people taxonomy and duplicate info.

## What does this Pull Request (PR) do?

This PR includes the correct path to the persName ID included in the personography file. The basic idea is to include ```ref="./Personography.xml#ID"``` instead of ```xml:id="#ID"```. As we'll be using a separate file rather than the taxonomy initially included in the template, I have removed the "people" taxonomy from the files (so Bryan is not completely confused). Since the personography file includes external IDs and birth and death dates, I have removed them from the sourceDesc/head section. This is redundant. Finally, it looks like we consistently leave out an "e" in Edgeworth's name in sourceDesc/head. I've fixed this throughout.

## How should this be tested?

One of the main reasons I opened this PR was to address invalid XML files. Ensure that each XML file has a green box. Please also check closely to make sure that I have not inadvertently changed the ID to refer to a different person. With both Mr. and Mrs. Miles appearing frequently, it's possible that I used the wrong code at some point.

## Other Considerations

I've noticed that sometimes we use <correspDesc> and sometimes we don't. We should streamline this (and also think through the larger issue of roleTerms) in the future.

## Interested Parties

@hilaryhavens @krame21 @elizawilcox
